apps/meteor/app/federation-v2/server/infrastructure/matrix/Bridge.ts

```diff
@@ -146,6 +146,33 @@ export class MatrixBridge implements IFederationBridge {
 		}
 	}
 
+	public async verifyInviteeId(externalInviteeId: string): Promise<string> {
```

Comment:
I believe it would be better to have an enum defining all the possibilities rather than having an arbitrary string, what do you think?

Reply:
Using enums now!

[apps/meteor/app/federation-v2/server/infrastructure/matrix/Bridge.ts](https://github.com/RocketChat/Rocket.Chat/pull/28096/files#diff-d16dc61cc516cc222f5b2e375e0e74a58bee770a4e1e190cb025d2970518d0b9)